### ISSUE TITLE
Prompted to let Brave control my computer with a11y features

### ISSUE THREAD

**Issue by tomlowenthal** [brave/brave-browser#3435]
Clicking play on a Twitter video led Brave to prompt me to enable a11y features. What's up with that?

![screen shot 2019-02-20 at 4 47 50 pm](https://user-images.githubusercontent.com/639937/53135266-70941a00-352f-11e9-92e9-e5776ff87db0.png)

**Comment by bsclifton** [comment]
I believe you brought this up on Slack and the root cause was found... @tomlowenthal was it because you're using LastPass? (and something in the extension is making a call which prompts that?)

**Comment by tomlowenthal** [comment]
I'm not using LastPass, so this must be a different issue.

**Comment by rebron** [comment]
macOS Mojave specific

**Comment by kjozwiak** [comment]
Moving to P2 as we're getting more folks asking us what this means and why it's being displayed.

**Comment by rebron** [comment]
Reproduced by hitting play on Soundcloud.com. @anthonypkeane

**Comment by anthonypkeane** [comment]
I had many tabs open, one being Soundcloud.
I upgraded to Version 0.61.52 
I navigated to Soundcloud, pressed play and it appeared

**Comment by themartorana** [comment]
Playing a twitter video appeared for me. Specifically, if it matters, this one, which is not YouTube but ABC News. https://twitter.com/brianbeutler/status/1111284475386839041

Browser: Version 0.61.51 Chromium: 73.0.3683.75 (Official Build) (64-bit)

**Comment by simonhong** [comment]
I also can reproduce this with 0.61 stable when I load https://www.twitch.tv/.
However, chrome stable(C74) doesn't show this dialog when loads https://www.twitch.tv/.
When brave play media, does it something special?

**Comment by kjozwiak** [comment]
Removing from milestone as there's no PR/work completed. As per https://github.com/brave/brave-browser/wiki/Triage-Guidelines, only `release/blockers` or completed work belong in milestones 👍

**Comment by bsclifton** [comment]
@simonhong this has been assigned to you, but I don't think you're actively looking at it. Should we unassign or did you want to try and find a fix?

**Comment by simonhong** [comment]
@bsclifton I tried to fix but it was difficult to me at that time. Please find other MacOS expert. :)

**Comment by bsclifton** [comment]
Bumping down to a P4 - this hasn't been a hugely impactful issue (not a lot of comments, etc) and we haven't prioritized work on it

**Comment by bsclifton** [comment]
Closing as stale - I haven't gotten this in quite some time. Let's re-open if we encounter it again 😄
